Makefile.kokkos

```diff
@@ -587,9 +587,9 @@ ifeq ($(KOKKOS_INTERNAL_USE_HPX), 0)
   ifeq ($(KOKKOS_INTERNAL_ENABLE_DEPRECATED_CODE), 1)
     tmp := $(call kokkos_append_header,"\#define KOKKOS_ENABLE_DEPRECATED_CODE")
   endif
-  ifeq ($(KOKKOS_INTERNAL_DISABLE_DEPRECATED_CODE), 0)
-    tmp := $(call kokkos_append_header,"\#define KOKKOS_ENABLE_DEPRECATED_CODE")
-  endif
+#  ifeq ($(KOKKOS_INTERNAL_DISABLE_DEPRECATED_CODE), 0)
+#    tmp := $(call kokkos_append_header,"\#define KOKKOS_ENABLE_DEPRECATED_CODE")
+#  endif
```

Comment:
Did you mean to commit this?

Reply:
I guess I can remove it outright. The option exists, but it doesn't do anything anymore since DISABLE_DEPRECATED_CODE is the default.